test/test_build_python.py

```diff
@@ -84,7 +84,7 @@ def test_build_package():
             src_base = Path(python_build_task.context.args.path)
 
             source_files_before = set(src_base.rglob('*'))
-            event_loop.run_until_complete(python_build_task.build())
+            assert not event_loop.run_until_complete(python_build_task.build())
```

Comment:
For readability please store the result in a local variable (e.g. `rc`) and then `assert` in a separate line.

Reply:
1f503c4